scapula/assets/js/views/Main.coffee

```diff
@@ -0,0 +1,33 @@
+define (require) ->
+  _BaseViews = require '_BaseViews'
+
+  SimulatorList = require 'collections/SimulatorList'
+  SimulatorListView = require 'views/SimulatorList'
+  SearchView = require 'views/Search'
+  SimulatorAddView = require 'views/SimulatorAdd'
+
+  class MainView extends _BaseViews.ParentView
+    template: require 'raw!../../templates/main.html'
+
+    afterRender: =>
+      simulatorList = new SimulatorList [
+        name: 'Alma'
+        type_number: 'A1000'
+        price: 25000
+      ,
+        name: 'Barack'
+        type_number: 'B1000'
+        price: 25000
+      ]
+
+      searchView = @createChild SearchView

```

Comment:
@createChild(SearchView, {}, 'search').$el.appendTo @$el

Lehet egy menetben is és akkor változó sem kell, a view-kat illik elnevezni, hogy utána könnyen lehessen őket @getChild-olni ha kell.